config/config.go

```diff
@@ -28,6 +28,7 @@ type KraftKit struct {
 	DefaultArch    string `yaml:"default_arch" env:"KRAFTKIT_DEFAULT_ARCH" usage:"The default architecture to use when invoking architecture-specific code" noattribute:"true"`
 	ContainerdAddr string `yaml:"containerd_addr,omitempty" env:"KRAFTKIT_CONTAINERD_ADDR" long:"containerd-addr" usage:"Address of containerd daemon socket" default:""`
 	EventsPidFile  string `yaml:"events_pidfile" env:"KRAFTKIT_EVENTS_PIDFILE" long:"events-pid-file" usage:"Events process ID used when running multiple unikernels" default:"/var/kraftkit/events.pid"`
+	ShowPreamble   bool   `yaml:"show_preamble" env:"KRAFTKIT_SHOW_PREAMBLE" long:"show-preamble" usage:"Show the QEMU preamble when running a unikernel" default:"false"`
```

Comment:
I think this can be turned into a turned into cmdfactory dynamically registered flag added to `init.go`

Reply:
please no more dynamic flags if not completely necessary

They don't always work for me so I don't want to rely on them

Reply:
That's why I started replacing/removing some of them

Reply:
```diff
diff --git a/machine/qemu/init.go b/machine/qemu/init.go
index c0b4186..7f890dd 100644
--- a/machine/qemu/init.go
+++ b/machine/qemu/init.go
@@ -4,7 +4,13 @@
 // You may not use this file except in compliance with the License.
 package qemu
 
-import "encoding/gob"
+import (
+       "encoding/gob"
+
+       "kraftkit.sh/cmdfactory"
+)
+
+var qemuShowSgaBiosPreamble string
 
 func init() {
        // Register only used supported interfaces later used for serialization.  To
@@ -468,4 +474,15 @@ func init() {
 
        // CLI configuration
        gob.Register(QemuConfig{})
+
+       // Register additional command-line arguments
+       cmdfactory.RegisterFlag(
+               "kraft run",
+               cmdfactory.StringVar(
+                       &qemuShowSgaBiosPreamble,
+                       "qemu-sgabios-preamble",
+                       "",
+                       "Show the QEMU SGABIOS preamble when running a unikernel",
+               ),
+       )
 }
 ```

Reply:
But yes, it would make sense, it's only related to running + logs

Reply:
when does it not work for you? rather we solve that than remove it. For me, the global config option doesn't make sense since it's so specific to this use case